ITEM.tsv

```diff
@@ -8523,261 +8523,261 @@ ITEM_20151102_008522	Unsent Letters
-ITEM_20151224_008694
-ITEM_20151224_008695
-ITEM_20151224_008696
-ITEM_20151224_008697
-ITEM_20151224_008698
-ITEM_20151224_008699
-ITEM_20151224_008700
-ITEM_20151224_008701
-ITEM_20151224_008702
-ITEM_20151224_008703
-ITEM_20151224_008704
-ITEM_20151224_008705
-ITEM_20151224_008706
-ITEM_20151224_008707
-ITEM_20151224_008708
-ITEM_20151224_008709
-ITEM_20151224_008710
-ITEM_20151224_008711
-ITEM_20151224_008712
-ITEM_20151224_008713
-ITEM_20151224_008714
-ITEM_20151224_008715
-ITEM_20151224_008716
-ITEM_20151224_008717
-ITEM_20151224_008718
-ITEM_20151224_008719
-ITEM_20151224_008720
-ITEM_20151224_008721
-ITEM_20151224_008722
-ITEM_20151224_008723
-ITEM_20151224_008724
-ITEM_20151224_008725
-ITEM_20151224_008726
-ITEM_20151224_008727
-ITEM_20151224_008728
-ITEM_20151224_008729
-ITEM_20151224_008730
-ITEM_20151224_008731
-ITEM_20151224_008732
-ITEM_20151224_008733
-ITEM_20151224_008734
-ITEM_20151224_008735
-ITEM_20151224_008736
-ITEM_20151224_008737
-ITEM_20151224_008738
-ITEM_20151224_008739
-ITEM_20151224_008740
-ITEM_20151224_008741
-ITEM_20151224_008742
-ITEM_20151224_008743
-ITEM_20151224_008744
-ITEM_20151224_008745
-ITEM_20151224_008746
-ITEM_20151224_008747
-ITEM_20151224_008748
-ITEM_20151224_008749
-ITEM_20151224_008750
-ITEM_20151224_008751
-ITEM_20151224_008752
-ITEM_20151224_008753
-ITEM_20151224_008754
-ITEM_20151224_008755
-ITEM_20151224_008756
-ITEM_20151224_008757
-ITEM_20151224_008758
-ITEM_20151224_008759
-ITEM_20151224_008760
-ITEM_20151224_008761
-ITEM_20151224_008762
-ITEM_20151224_008763
-ITEM_20151224_008764
-ITEM_20151224_008765
-ITEM_20151224_008766
-ITEM_20151224_008767
-ITEM_20151224_008768
-ITEM_20151224_008769
-ITEM_20151224_008770
-ITEM_20151224_008771
-ITEM_20151224_008772
-ITEM_20151224_008773
-ITEM_20151224_008774
-ITEM_20151224_008775
-ITEM_20151224_008776
-ITEM_20151224_008777
-ITEM_20151224_008778
-ITEM_20151224_008779
-ITEM_20151224_008780
+ITEM_20151224_008526	Beginning of Savior Armband
+ITEM_20151224_008527	Chief Armband
+ITEM_20151224_008528	Silver Armband
+ITEM_20151224_008529	Dia Armbad
+ITEM_20151224_008530	Alcohol Decoration Armband
+ITEM_20151224_008531	Shaman Armband
+ITEM_20151224_008532	Orient Armband
+ITEM_20151224_008533	Golden Armband
+ITEM_20151224_008534	Leaf Pattern Armband
+ITEM_20151224_008535	Dragonfly Armband
+ITEM_20151224_008536	Armband with Feelings
+ITEM_20151224_008537	No_hair
+ITEM_20151224_008538	Terius
+ITEM_20151224_008539	Hair [Male]
+ITEM_20151224_008540	Equipping this will change your hair.
+ITEM_20151224_008541	Dandy Wave Curl
+ITEM_20151224_008542	Curvy Wave Twintail
+ITEM_20151224_008543	Hair [Female]
+ITEM_20151224_008544	Curvy Wave
+ITEM_20151224_008545	Koperon Sword
+ITEM_20151224_008546	Superior Koperon Sword
+ITEM_20151224_008547	Dark Sword
+ITEM_20151224_008548	Soldier's Falchion
+ITEM_20151224_008549	Twin Blade
+ITEM_20151224_008550	Superior Twin Blade
+ITEM_20151224_008551	 - 2% chance to increase magic defense for 15 seconds after receiving magic damage
```

Comment:
You could create the sentence like that, but I guess it is better to go with the format that's been already used, which is to put the effect in brackets:

- [Magic Defense Increase] 2% chance to increase magic defense for 15 seconds after receiving magic damage